src/setup-docker/templates/fabric-config/configtx-raft-template.yaml.ejs

```diff
@@ -1,8 +1,8 @@
-<% if (ordererOrg.consensus == 'etcdraft') { %>
+<% if (ordererGroup.consensus == 'etcdraft') { %>
     EtcdRaft:
-        Consenters:<% ordererOrg.orderers.forEach(function(orderer) { %>
+        Consenters:<% ordererGroup.orderers.forEach(function(orderer) { %>
             - Host: <%= orderer.address %>
               Port: <%= orderer.port %>
-              ClientTLSCert: crypto-config/ordererOrganizations/<%= orderer.domain %>/orderers/<%= orderer.address %>/tls/server.crt
-              ServerTLSCert: crypto-config/ordererOrganizations/<%= orderer.domain %>/orderers/<%= orderer.address %>/tls/server.crt
+              ClientTLSCert: crypto-config/peerOrganizations/<%= orderer.domain %>/peers/<%= orderer.address %>/tls/server.crt
```

Comment:
maybe it is just `organizations`? or `orgs`? or just `crypto-config/<%= orderer.domain %>/...`